boards/arm/usb_kw24d512/doc/usb_kw24d512.rst

```diff
@@ -0,0 +1,127 @@
+.. _usb_kw24d512:
+
+NXP USB-KW24D512
+################
+
+Overview
+********
+
+The USB-KW24D512 board is an IEEE 802.15.4 packet sniffer in a
+convenient USB dongle form factor.
+
+- Packet sniffers are essential for wireless network development allowing
+  over-the-air traffic monitoring
+- As a standalone board, the USB-KW24D512 can also be used in a lab as a
+  development platform and programmed as a network node (for example)
+
+Hardware
+********
+
+- Kinetis KW2xD-2.4 GHz 802.15.4 Wireless Radio Microcontroller
+  (50 MHz, 512 KB flash memory, 64 KB RAM, low-power, crystal-less USB)
+- USB Type A Connector
+- Two blue LEDs
+- One user push button
+- One reset button
+- Integrated PCB Folded F-type antenna
+- Cortex 10-pin (0.05”) JTAG debug port for target MCU
+
+For more information about the KW2xD SiP and USB-KW24D512 board:
```

Comment:
suggest spelling out SiP in this first use:

     KW2xD System-in-Package (SiP)